terminal/bundles/org.eclipse.terminal.connector.local/src/org/eclipse/terminal/connector/local/launcher/LocalLauncherDelegate.java

```diff
@@ -108,7 +108,7 @@ public void execute(Map<String, Object> properties, ITerminalService.Done done)
 		// Initialize the local terminal working directory.
 		if (!properties.containsKey(ITerminalsConnectorConstants.PROP_PROCESS_WORKING_DIR)) {
 			// By default, start the local terminal in the users home directory
-			String initialCwd = IPreferenceKeys.getPreferences()
+			String initialCwd = UIPlugin.getScopedPreferences()
```

Comment:
This is not the same! Previously the preferences of `org.eclipse.terminal.view.ui` where used, now `org.eclipse.terminal.connector.local` is used, what likely will not work!

The problem is that the terminal.view has the preference page to configure some aspects here that should be reused.

Reply:
Good catch, thanks!